lib/replica_pools/version.rb

```diff
@@ -1,3 +1,3 @@
 module ReplicaPools
-  VERSION = "2.6.0"
+  VERSION = "2.6.1"
```

Comment:
This feels like a patch-y change to me, but let me know if I should be updating the minor version instead!